tests/test_plugin_rtpplay.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from streamlink.plugins.eurocom import Eurocom
+
+
+class TestPluginEurocom(unittest.TestCase):
+    def test_can_handle_url(self):
+        # should match
+        self.assertTrue(Eurocom.can_handle_url("http://www.rtp.pt/play/"))
```

Comment:
Thank you.. Hindsight is 20/20, I limited my self to copy pasta it without paying much attention.